src/sql/base/browser/ui/panel/media/panel.css

```diff
@@ -110,26 +101,26 @@ panel {
 	flex-direction: row;
 }
 
-.tabbedPanel.vertical > .title {
+.tabbedPanel.vertical>.title {
```

Comment:
Why are you removing the spaces? Makes it harder to read and isn't the standard style for CSS